apigee-test-shared/test.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+BIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT="$( cd "$BIN_DIR/.." && pwd)"
+
+main()
+{
+    cd "$ROOT"
+
+    npm test || die "Mocha test failed"
+    echo "All tests pass!"
+}
+
+die()
+{
+    echo "ERROR: $*"
+    exit 1
+}
+
+main "$@"
```

Comment:
This bash file looks like something I would've written.

![chef finger kiss](https://media.giphy.com/media/lGV8Xw3pCyB20/giphy.gif)

Reply:
Credit Rachel and Ramya!

Reply:
lol, you did write it @zandersmith-wp! This was stolen from the wpe-generator test script!